pipelineruns/integration_resolver_pipelinerun_pass.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: integration-resolver-pipelinerun-
+  annotations:
+    pipelinesascode.tekton.dev/on-target-branch: '[main]'
+    pipelinesascode.tekton.dev/repo-url: 'https://github.com/redhat-appstudio/build-service'
+    pipelinesascode.tekton.dev/sha-title: 'Merge pull request #35 from psturc/PLNSRVCE-281-vol2'
+    pipelinesascode.tekton.dev/git-auth-secret: pac-gitauth-nthx
+    pipelinesascode.tekton.dev/max-keep-runs: '5'
+    pipelinesascode.tekton.dev/sha-url: https://github.com/redhat-appstudio/build-service/commit/f13b881e3404ad3401fa7b533df3f7daddd11b55
+    pipelinesascode.tekton.dev/on-event: '[push]'
+    pipelinesascode.tekton.dev/installation-id: '23331085'
+  labels:
+    pipelines.appstudio.openshift.io/type: "test"
+    test.appstudio.openshift.io/test: "component"
+    appstudio.openshift.io/snapshot: "snapshot-sample"
+    appstudio.openshift.io/component: "component-sample"
+    appstudio.openshift.io/application: "application-sample"
+    test.appstudio.openshift.io/scenario: "integrationtestscenario-sample1"
+    pipelinesascode.tekton.dev/event-type: push
+    pipelinesascode.tekton.dev/state: completed
+    pipelinesascode.tekton.dev/sender: psturc
+    pipelinesascode.tekton.dev/check-run-id: '7108761088'
+    pipelinesascode.tekton.dev/branch: refs-heads-main
+    pipelinesascode.tekton.dev/url-org: redhat-appstudio
+    pipelinesascode.tekton.dev/original-prname: build-service-on-push
+    pipelinesascode.tekton.dev/url-repository: build-service
+    pipelinesascode.tekton.dev/repository: build-service-pac
+    pipelinesascode.tekton.dev/sha: f13b881e3404ad3401fa7b533df3f7daddd11b55
+    pipelinesascode.tekton.dev/git-provider: github
+spec:
+  pipelineRef:
+    resolver: git
+    params:
+    - name: url 
+      value: http://github.com/redhat-appstudio/integration-examples.git 
+    - name: revision
+      value: tekton_resolver
```

Comment:
Do we want to set this to `main` so we can update the pipeline definition in the future without needing to update the ITS too?

Reply:
Done